Mathlib/Tactic/DeriveTraversable.lean

```diff
@@ -4,6 +4,9 @@ Released under Apache 2.0 license as described in the file LICENSE.
 Authors: Simon Hudon
 -/
 import Mathlib.Control.Traversable.Lemmas
+import Lean.Elab.Match
```

Comment:
I'm assuming these were transitively imported before?

Reply:
Yeah, this is the result of shake doing its thing